src/ngraph/frontend/onnx_import/core/graph.cpp

```diff
@@ -25,26 +26,29 @@ namespace ngraph
     {
         namespace detail
         {
-            std::string to_string(const std::set<std::string>& set)
+            std::string to_string(
+                const std::map<std::string, std::reference_wrapper<const onnx::NodeProto>>& map)
             {
                 std::string result;
-                for (auto it = std::begin(set); it != std::end(set); ++it)
+                for (auto it = std::begin(map); it != std::end(map); ++it)
                 {
-                    result += (it != std::begin(set) ? ", " : "") + *it;
+                    result += (it != std::begin(map) ? ", " : "") + it->first;
                 }
                 return result;
             }
 
-            inline std::string to_string(const onnx::NodeProto& node_proto)
+            std::string get_node_domain(const onnx::NodeProto& node_proto)
             {
-                return (node_proto.domain().empty() ? "" : node_proto.domain() + ".") +
-                       node_proto.op_type();
+                return (node_proto.domain().empty() ? "" : node_proto.domain());
             }
-        }
 
-        Graph::Graph(const onnx::GraphProto& graph_proto,
-                     const Model& model,
-                     const Weights& weights)
+            std::string to_string(const onnx::NodeProto& node_proto)
```

Comment:
Actually your proposition seems to me a bit confusing - it's hard to understand what `reverse_dns` part relates to without reading the doc you linked. In fact the `reverse_dns` is only a form of operators domain. 

The operator is uniquely identified by the tuple (_domain_, _op\_type_, _since\_version_) https://github.com/onnx/onnx/blob/2e7099ee7c37b196c197c9a084a97698a41da232/onnx/onnx-operators.in.proto#L74. At this point we don't have _since\_version_ information, thus I would rename it to `get_op_uid` since we use the resulting value as a operator unique identificator.